### ISSUE TITLE
Query regarding paid plan for IEX cloud

### ISSUE THREAD

**Issue by yccheok** [iexg/IEX-API#694]
1) Currently, we are using web socket (https://iextrading.com/developer/docs/#websockets) to access real-time stock price info. We do not subscribe to any paid plan in https://iexcloud.io/pricing/ Going forward, can we still use https://iextrading.com/developer/docs/#websockets at no cost, or we need to start subscribe to paid plan listed in https://iexcloud.io/pricing/  ?

2) Currently, we are accessing dividend data in https://api.iextrading.com/1.0/stock/mcd/dividends/5y at no cost. However, as you can see, the data is pretty outdated. The last dividend data is Feb 2018. Does that mean, we need to subscribe https://iexcloud.io/pricing/ to get better data? Thanks!

In short, may I know, going forward, will the previously free API listed in https://iextrading.com/developer/docs/#version-1-0 still be supported, or we need to migrate to paid Version 2 API ?

**Comment by evancarter-iex** [comment]
1) You will need to subscribe to a paid plan for streaming, we are switching to SSE (Server Sent Events). See here for more info: https://en.wikipedia.org/wiki/Server-sent_events

2) Yes our data in IEXCloud is much improved over our current data provider.  You can sample some of the data with the free version.

The old API will remain for IEX only data. Though you can move over to IEXcloud is since it will be free there as well. See https://iexcloud.io/docs/api/#tops for more information.

**Comment by yccheok** [comment]
Thanks, and that's fair enough.

However, we do have some concern regarding the duration given for migration.

From https://github.com/iexg/IEX-API/issues/557 , I understand IEX plan to retire free V1 API, and urge all developers move to to paid V2 API, by May 2019.

Since, May 2019 is pretty short notice period, for us to perform all necessary code migration and testing, would IEX consider to continue support free V1 API, till next year?

I raise the same concern at https://github.com/iexg/IEX-API/issues/655

Thank you.

**Comment by joshuablackburn-iex** [comment]
Non IEX data will be available until June 1st 2019.

**Comment by joshuablackburn-iex** [comment]
We have built IEX Cloud to be as seamless as possible to switch from v1. Most, if not all, endpoints should work by just switching the base url.
